lib/tasks/migrate_to_pg.rake

```diff
@@ -118,13 +128,36 @@ def migrate_fulfillment(pt_model, mongo_pt_model, pg_model, mongo_model, relatio
     if mongo_obj['created_by_id'].present?
       pg_obj.versions.first.update whodunnit: User.find_by(mongo_id: mongo_obj['created_by_id'].to_s)&.id
     end
+  end
 
-    # Check
-    if pg_model.count != mongo_model.count
-      raise 'PG and mongo counts are in disagreement, aborting'
+  # Check
+  if pg_model.count != Patient.count
+    raise 'Every patient not associated with only one fulfillment'
+  end
+
+  puts "#{pg_model.count} Fulfillment migrated to pg"
+end
+
+def migrate_submodel(pt_model, mongo_pt_model, pg_model, mongo_model, relation, parent_relation, transform)
+  attributes = pg_model.attribute_names
+  mongo_pt_model.collection.find.batch_size(100).each do |doc|
+    mongo_objs = doc[relation] || []
+    mongo_objs.each do |obj|
+      pg_attrs = obj.slice(*attributes)
+      pg_attrs = transform.call(pg_attrs, obj, doc)
+      pg_obj = pg_model.create! pg_attrs
```

Comment:
Technically, we could DRY this out a bit by combining with the guts of `migrate_fulfillment`, but I think I prefer the readability hear in our already complicated, one-time use, super scary database conversion script.

Reply:
yeah, same. the first draft was DRY-er and it made it significantly harder to review.